cert/cert.go

```diff
@@ -95,7 +96,8 @@ var NodeCSRTempl string = `{
     "hosts": [
       "{{.Name}}",
       "{{.LongName}}",
-      "{{.Fqdn}}"
+      "{{.Fqdn}}",
+	  {{.SubjectAltNames}}
```

Comment:
instead of doing a json marshalling, let's have a template range function `range san := .SANs`
(ps, also there is a mix of tab and spaces)